### ISSUE TITLE
Edit Link data type settings fails

### ISSUE THREAD

**Issue by nicpitsch** [2sic/2sxc#1230]
**I'm submitting a ...**  
```
- [x] bug report 
```

**...about**  
```
- [x] admin experience UI
```

**Current behavior**
Attempt to edit settings of data type *Link* causes "Object reference not set to an instance of an object" error.

**Expected behavior**
Settings Form is opening.

**Minimal reproduction of the problem with instructions**
* Create a *Link* data type and try to edit it's settings. 
* I suppose the issue is only relevant for a fresh DNN 9 (see *Environment* below).

**Environment:**
* A fresh DNN 09.01.01 (not upgraded from version < 9).
* Latest 2sxc version 9.3.0.

**Further info, supposed reasons:**
* A new DNN 09.01.01 does not contain "RadEditor Manager" Provider anymore (not as installed Provider Extension nor as available Provider Extension).
* Admin Error Log:
```
AbsoluteURL:/desktopmodules/2sxc/api/eav/entities/getmanyforediting
DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke
ExceptionGUID:daeea0c1-3c0e-4d3a-af60-ec6c1238a982
AssemblyVersion:
PortalId:-1
UserId:-1
TabId:-1
RawUrl:
Referrer:
UserAgent:
ExceptionHash:dq8HW6iNWqxroJ7rxNheAOY8M3k=
Message:Object reference not set to an instance of an object.
StackTrace:
at ToSic.Eav.WebApi.EntitiesController.GetManyForEditing(Int32 appId, List`1 items) in C:\Projects\eav-server\ToSic.Eav.WebApi\EntitiesController.cs:line 154 at ToSic.SexyContent.WebApi.EavApiProxies.EntitiesController.GetManyForEditing(List`1 items, Int32 appId) in C:\Projects\2SexyContent\Web\DesktopModules\ToSIC_SexyContent\Sxc WebApi\EavApiProxies\EntitiesController.cs:line 95 at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.b__9(Object instance, Object[] methodParameters) at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken) --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Tracing.ITraceWriterExtensions.d__18`1.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Controllers.ApiControllerActionInvoker.d__0.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Tracing.ITraceWriterExtensions.d__18`1.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Controllers.ActionFilterResult.d__2.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Filters.AuthorizationFilterAttribute.d__2.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Filters.AuthorizationFilterAttribute.d__2.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Filters.AuthorizationFilterAttribute.d__2.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Filters.AuthorizationFilterAttribute.d__2.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at System.Web.Http.Controllers.ExceptionFilterResult.d__0.MoveNext()
InnerMessage:
InnerStackTrace:
Source:ToSic.Eav.WebApi
FileName:
FileLineNumber:0
FileColumnNumber:0
Method:
```

**Comment by nicpitsch** [comment]
Completion:

- The Telerik RadEditorProvider for DNN 9.1.x is available [here](https://github.com/dnnsoftware/Dnn.Platform/releases) as a separate installation,

**Comment by iJungleboy** [comment]
thx, will look into it.

**Comment by iJungleboy** [comment]
fixed in 9.3 Prerelease 3 (just released now)
